packages/lesswrong/server/notificationCallbacks.tsx

```diff
@@ -125,7 +131,13 @@ const getNotificationTiming = (typeSettings): DebouncerTiming => {
   }
 }
 
-const createNotification = async (userId: string, notificationType: string, documentType: string|null, documentId: string|null) => {
+const createNotification = async ({userId, notificationType, documentType, documentId, noEmail}:{
+    userId: string, 
+    notificationType: string, 
+    documentType: string|null, 
+    documentId: string|null, 
+    noEmail?: boolean
```

Comment:
@jimrandomh, should documentType/documentId be converted into optional parameters here now, and remove the "null" type option?